code/game/turfs/simulated/outdoors/outdoors_attackby.dm

```diff
@@ -27,29 +27,29 @@
 	. = ..()
 
 /turf/simulated/floor/outdoors/attackby(obj/item/C, mob/user)
-
-	if(can_dig && istype(C, /obj/item/shovel))
-		to_chat(user, SPAN_NOTICE("\The [user] begins digging into \the [src] with \the [C]."))
-		var/delay = (3 SECONDS * C.toolspeed)
-		user.setClickCooldown(delay)
-		if(do_after(user, delay, src))
-			if(!(locate(/obj/machinery/portable_atmospherics/hydroponics/soil) in contents))
-				var/obj/machinery/portable_atmospherics/hydroponics/soil/soil = new(src)
-				user.visible_message(SPAN_NOTICE("\The [user] digs \a [soil] into \the [src]."))
-			else
-				var/loot_type = get_loot_type()
-				if(loot_type)
-					loot_count--
-					var/obj/item/loot = new loot_type(src)
-					to_chat(user, SPAN_NOTICE("You dug up \a [loot]!"))
+	if (!snow_layers) // If we have snow, fall back to the base attackby() that has snow clearing logic
```

Comment:
Seeding a mote of an idea: `snow_layers = -1` as meaning that a turf _can't_ have snow. Would mean more work, but could be useful in the future.